UI/homepage.py

```diff
@@ -62,20 +142,55 @@ def draw_filter_search_screen():
     current_screen = 'filter_search'
     update_items_display()
 
+def draw_recipe_search_screen():
+    global current_screen, back_button, recipe_title, recipe_search_button, filter_buttons
```

Comment:
`back_button` and `current_screen` are set to global status here, but they've already been initialized as global on line 122